Makefile

```diff
@@ -129,10 +130,15 @@ lint-deps:
 		set -ex; curl -fsSL https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.$$(uname).x86_64.tar.xz | tar -xJv --strip-components=1 shellcheck-stable/shellcheck; \
 		mv shellcheck $(shell go env GOPATH)/bin/shellcheck; chmod +x $(shell go env GOPATH)/bin/shellcheck; \
 	fi
+	@if ! which gosec >/dev/null || [[ "$$(gosec --version)" != *${GOSEC_VERSION}* ]]; then \
+		set -o pipefail; \
+		go install github.com/securego/gosec/v2/cmd/gosec@${GOSEC_VERSION}; \
+	fi
 
 .PHONY: lint
 lint: lint-deps
 	golangci-lint run
+	gosec -conf .gosec.json ./...
```

Comment:
Do we want to add `-version=${GOSEC_VERSION}` here for cases where multiple versions of gosec are installed?